<launch>
  <arg name="relay_kinect" default="true"/>
  <arg name="camera" default="kinect_head" unless="$(arg relay_kinect)"/>
  <arg name="camera" default="kinect_head_remote" if="$(arg relay_kinect)"/>

  <include file="$(find jsk_diabolo_pr2)/launch/relay_kinect.launch"
           if="$(arg relay_kinect)"/>
  
  <node name="tf_transform_cloud" pkg="jsk_pcl_ros" type="tf_transform_cloud">
    <remap from="~input" to="$(arg camera)/depth_registered/points" />
    <param name="~target_frame_id" value="base_footprint"/>
  </node>

  <node name="calc_idle_diabolo_state" pkg="jsk_diabolo_pr2" type="calc_idle_diabolo_state"/>
</launch>
